### ISSUE TITLE
About point depth used in DepthLSSTransform

### ISSUE THREAD

**Issue by chenchaoxu** [mit-han-lab/bevfusion#90]
Hi talents,

I have a question about DepthLSSTransform used in your bevFusion model. It seems that info of point depth is used for LSS. Dose it get large gain of performance comparing to the orginal LSS? Plus, why not chosing to set point depth as surpervision instead?

**Comment by kentang-mit** [comment]
Hello @chenchaoxu,

Thanks for your interest. We did not notice a large gain of performance comparing to the original LSS and this is the reason why we did not talk about it in the paper. Your suggestion is actually a very good idea, we definitely want to try that in the future. We have tried directly converting the 2.5D depth to a one-hot representation (so that you do not have to learn it and can directly use the "GT"), but I remember that did not work out. 

Best,
Haotian
